perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh

```diff
@@ -42,22 +42,19 @@ sudo dpkg -i $HOME/release/packages/gcsfuse_${GCSFUSE_VERSION}_amd64.deb
 
 # Mounting gcs bucket
 cd "./perfmetrics/scripts/"
-echo "Mounting gcs bucket"
-mkdir -p gcs
-LOG_FILE=${KOKORO_ARTIFACTS_DIR}/gcsfuse-logs.txt
-GCSFUSE_FLAGS="--implicit-dirs --max-conns-per-host 100 --enable-storage-client-library --debug_fuse --debug_gcs --log-file $LOG_FILE --log-format \"text\" --stackdriver-export-interval=30s"
-BUCKET_NAME=periodic-perf-tests
-MOUNT_POINT=gcs
-# The VM will itself exit if the gcsfuse mount fails.
-gcsfuse $GCSFUSE_FLAGS $BUCKET_NAME $MOUNT_POINT
+
+LOG_FILE_PERIODIC_PERF_TESTS=gcsfuse-logs.txt
+LOG_FILE=${KOKORO_ARTIFACTS_DIR}/${LOG_FILE_PERIODIC_PERF_TESTS}
+GCSFUSE_FLAGS_PERIODIC_PERF_TESTS="--implicit-dirs --max-conns-per-host 100 --enable-storage-client-library --debug_fuse --debug_gcs --log-file $LOG_FILE --log-format \"text\" --stackdriver-export-interval=30s"
 
 # Executing perf tests
 chmod +x run_load_test_and_fetch_metrics.sh
-./run_load_test_and_fetch_metrics.sh
-
-sudo umount $MOUNT_POINT
+./run_load_test_and_fetch_metrics.sh "$GCSFUSE_FLAGS_PERIODIC_PERF_TESTS"
 
-# ls_metrics test. This test does gcsfuse mount first and then do the testing.
+# ls_metrics test. This test does gcsfuse mount with the passed flags first and then does the testing.
+LOG_FILE_LIST_TESTS=gcsfuse-list-tests-logs.txt
+LOG_FILE="$LOG_FILE_LIST_TESTS"
+GCSFUSE_FLAGS_LIST_TESTS="--implicit-dirs --max-conns-per-host 100 --enable-storage-client-library --debug_fuse --debug_gcs --log-file $LOG_FILE --log-format \"text\" --stackdriver-export-interval=30s"
```

Comment:
Given flags are same in both cases except for log file, can you check if we create single string for flags and substitute different log files in that string for read/write and list tests ? This will reduce maintaining two different strings.

Reply:
Sure!